### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#15785]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

- Operating System: all
- Browser: all
- Screenreading device: any
- Server destination: staging & production

## Steps to Recreate

1. Enter [https://www.va.gov/find-forms/] in browser
2. Have developer tools open, and the axe browser extension loaded
3. Run an axe audit
4. Verify that [SGLI, VGLI, and other VA life insurance programs] is called out as an error of "Heading levels should only increase by one"
5. This error is repeated throughout the component

### Sample 

**Current code**
```html
<h4 class="va-nav-linkslist-title">SGLI, VGLI, and other VA life insurance programs</h4>
```

**Recommended code**
```html
<h3 class="vads-u-font-size--h4">SGLI, VGLI, and other VA life insurance programs</h3>
```

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

## Screenshots

![Screen Shot 2020-11-04 at 12 05 42 PM](https://user-images.githubusercontent.com/57469/98143624-5337e880-1e97-11eb-962f-3c0f5f4c37e1.png)

**Comment by MarciMcGuireGCIO** [comment]
Assigning to new Decision Tools team; un-assigning from Public Websites

**Comment by zacharymorel** [comment]
Thread to current collaboration to determine best solution => https://dsva.slack.com/archives/C01DWJFD45D/p1605813270016600

**Comment by zacharymorel** [comment]
Good afternoon  @jenstrickland this is ready to be validated on https://staging.va.gov/ .

**Comment by jenstrickland** [comment]
@zacharymorel Thanks! Validated 🎉
